examples/Cargo.toml

```diff
@@ -31,6 +31,11 @@ serde_json = "1.0"
 bytes = "1.1"
 lazy_static = "1.4"
 rand = "0.8"
+# encode-microphone
+cpal = "0.14.0"
```

Comment:
I would drop the patch and stick to major and minor like most of the other lines